### ISSUE TITLE
Replace deprecated Widgets.

### ISSUE THREAD

**Issue by ManasviPatidar** [mdg-iitr/codephile-mobile#55]
Widgets like [RaisedButton](https://api.flutter.dev/flutter/material/RaisedButton-class.html), [FlatButton ](https://api.flutter.dev/flutter/material/FlatButton-class.html)are now deprecated.

- [ ] Identify all such Widgets
- [ ] Find and apply suitable replacements

**Comment by raghavpawar** [comment]
Hi, I would like to work on this issue.

**Comment by raghavpawar** [comment]
Hi, I have solved this issue. Will create a new PR once the previous one is merged.
